pkgs/tools/misc/todo/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, stdenv
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "todo";
+  version = "2.4";
+
+  src = fetchFromGitHub {
+    owner = "sioodmy";
+    repo = pname;
+    rev = "${version}";
```

Comment:
For the sake of readability I think its a bit nicer not to use `pname` here. That would also avoid unnecessarily depending on it which isnt a huge deal but we might as well. Also, since `rev` is already a string we dont need to interpolate it.

```suggestion
    repo = "todo";
    rev = version
```